backend/src/external/aws/lambdas/authenticate.py

```diff
@@ -1,34 +1,28 @@
 import logging
 import os
 
-import boto3
-import jwt
+from src import context
+from src.external.aws.iam import Iam
+from src.external.aws.secret_manager import SecretManager
+from src.login.authenticate import generate_role, has_valid_token
+from src.login.user.role import Role
+from src.secret.cloud_secret_manager import CloudSecretManager
+
+appContext = context.ApplicationContext()
+appContext.register(CloudSecretManager, SecretManager())
+appContext.register(Role, Iam)
 
-client = boto3.client("secretsmanager")
 ENVIRONMENT = os.environ["ENVIRONMENT"]
 
 
 def lambda_handler(event, context):
-    logging.info("Getting public key from AWS Secrets Manager")
-    public_key = client.get_secret_value(SecretId=f"document-extractor-{ENVIRONMENT}-public-key")["SecretString"]
-
     token = event["authorizationToken"].replace("Bearer ", "")
+    logging.info("Getting public key from AWS Secrets Manager")
```

Comment:
This line can be deleted because we are not about tog et the public key from AWS Secrets Manager.